phoenix-core-client/src/main/java/org/apache/phoenix/jdbc/PhoenixEmbeddedDriver.java

```diff
@@ -65,6 +65,7 @@ public abstract class PhoenixEmbeddedDriver implements Driver, SQLCloseable {
     private final static DriverPropertyInfo[] EMPTY_INFO = new DriverPropertyInfo[0];
     public final static String MAJOR_VERSION_PROP = "DriverMajorVersion";
     public final static String MINOR_VERSION_PROP = "DriverMinorVersion";
+    public final static String PATCH_VERSION_PROP = "DriverPatchVersion";
```

Comment:
This is not used anywhere.
Do we need this ?

Reply:
Nope, i just forgot to add that to the commit